app.py

```diff
@@ -157,7 +223,7 @@ def version_request():
 if __name__ == '__main__':
     UpdateQueueFromFile()
     scheduler = BackgroundScheduler()
-    scheduler.add_job(func=UpdateCurrentTimer, trigger="interval", seconds=60)
+    scheduler.add_job(func=UpdateCurrentTimer, trigger="interval", seconds=1)
```

Comment:
Maybe its better to set this to 10 seconds or so, otherwise there are a lot of file writes

Reply:
Instead, I opted to only write to the file when changes were applied
e31c45ec6961759e3c6d873254ffbe6c2acad14b